PC/pyconfig.h

```diff
@@ -76,6 +76,10 @@ WIN32 is still required for the locale module.
 #define MS_WINDOWS_NON_DESKTOP
 #endif
 
+#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY == WINAPI_FAMILY_GAMES)
+#define MS_XBOX
+#endif
```

Comment:
How about `MS_WINDOWS_GAMES`? They went out of their way to keep the API families and family partitions fairly generic in "winapifamily.h". Eventually something other than Xbox could use the `WINAPI_FAMILY_GAMES` API.

Reply:
I updated the name